main.tf

```diff
@@ -108,7 +108,7 @@ module "database" {
 }
 
 module "operator" {
-  source = "github.com/MaterializeInc/terraform-helm-materialize?ref=v0.1.8"
+  source = "github.com/MaterializeInc/terraform-helm-materialize?ref=v0.1.9"
```

Comment:
I also bump this in my PR https://github.com/MaterializeInc/terraform-aws-materialize/pull/49